dev-libs/libsodium/libsodium-1.0.16.ebuild

```diff
@@ -21,11 +21,14 @@ src_prepare() {
 	eautoreconf
 }
 
-src_configure() {
+multilib_src_configure() {
+	local ECONF_SOURCE="${S}"
 	local myconf
 
 	# --disable-pie needed on x86, bug #512734
-	use x86 && myconf="${myconf} --disable-pie"
+	if [[ ${ABI} == "x86" ]]; then
```

Comment:
I'm sorry but it just occurred to me that `${MULTILIB_ABI_FLAG}` would actually be better here as `${ABI}` differs e.g. between Linux and FreeBSD while the flag-var always will have `abi_x86_32`.

Reply:
Okay, will be changed. Should I always use MULTILIB_ABI_FLAG?

Reply:
Inside `multilib_*` phases, yes. Outside that it's not defined ;-).